gmso/tests/test_potential_templates.py

```diff
@@ -0,0 +1,129 @@
+import os
+import glob
+import pytest
+
+import sympy
+
+from gmso.lib.potential_templates import PotentialTemplates, JSON_DIR
+from gmso.exceptions import GMSOError
+from gmso.tests.base_test import BaseTest
+
+
+class TestPotentialTemplates(BaseTest):
+    @pytest.fixture
+    def templates(self):
+        return PotentialTemplates()
+
+    def test_singleton_behavior(self, templates):
+        assert id(templates) == id(PotentialTemplates())
+        assert id(PotentialTemplates()) == id(PotentialTemplates())
+
+    def test_lennard_jones_potential(self, templates):
+        lennard_jones_potential = templates['LennardJonesPotential']
+        assert lennard_jones_potential.name == 'LennardJonesPotential'
+        assert lennard_jones_potential.expression == sympy.sympify('4*epsilon*((sigma/r)**12 - (sigma/r)**6)')
+        assert lennard_jones_potential.independent_variables == {sympy.sympify('r')}
+
+    def test_mie_potential(self, templates):
+        mie_potential = templates['MiePotential']
+        assert mie_potential.name == 'MiePotential'
+        assert mie_potential.expression == sympy.sympify(
+            '(n/(n-m)) * (n/m)**(m/(n-m)) * epsilon * ((sigma/r)**n - (sigma/r)**m)')
+        assert mie_potential.independent_variables == {sympy.sympify('r')}
+
+    def test_opls_torsion_potential(self, templates):
+        opls_torsion_potential = templates['OPLSTorsionPotential']
+        assert opls_torsion_potential.name == 'OPLSTorsionPotential'
+        assert opls_torsion_potential.expression == sympy.sympify('0.5 * k0 + 0.5 * k1 * (1 + cos(phi)) + '
+                                                                  '0.5 * k2 * (1 - cos(2*phi)) +'
```

Comment:
could we make these lines a bit more similar either with the `+` at the end of each str, or at the beginning?